protocol/messenger_contact_requests_test.go

```diff
@@ -62,12 +62,7 @@ func (s *MessengerContactRequestSuite) newMessenger(shh types.Waku) *Messenger {
 }
 
 func (s *MessengerContactRequestSuite) findFirstByContentType(messages []*common.Message, contentType protobuf.ChatMessage_ContentType) *common.Message {
```

Comment:
Is this function still needed? It seems like we can just call `FindFirstByContentType`